packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts

```diff
@@ -188,23 +188,15 @@ export interface Node {
   extras?: Extras;
 }
 
-export interface ExternalImage {
```

Comment:
Combining the two glTF image types because I didn't feel the type complexity was doing anything to help us.